arrow/arrow.py

```diff
@@ -1258,6 +1258,13 @@ def humanize(
                 return locale.describe(granularity, delta, only_distance=only_distance)
 
             else:
+
+                if not granularity:
+                    raise ValueError(
+                        "Empty granularity list provided. "
+                        "Please select between 'second', 'minute', 'hour', 'day', 'week', 'month' or 'year'."
```

Comment:
The second sentence should be reworded since this is a list. It's currently phrased as if only one choice can be made.